src/renderer/viz/expressions/classification/Classifier.js

```diff
@@ -67,14 +66,17 @@ export default class Classifier extends BaseExpression {
     }
 
     _validateInputIsNumericProperty () {
-        checkInstance(this.expressionName, 'input', 0, Property, this.input);
         checkType(this.expressionName, 'input', 0, 'number', this.input);
     }
 
     toString () {
         return `${this.expressionName}(${this.input.toString()}, ${this.numCategories})`;
     }
 
+    get value () {
+        return this.breakpoints.map(br => br.expr);
```

Comment:
Does it make sense to reuse inside the current `getBreakpointList` method?

Reply:
This is actually something we're not using anywhere right now, and we can even remove it. This is a result of a quick test of values / eval I was doing, and as a result I opened this issue, cause the truth is that `value` getters are not always working as expected in every expression https://github.com/CartoDB/carto-vl/issues/1288